pillar.example

```diff
@@ -1,5 +1,5 @@
 elasticsearch:
-  major_version: 2
+  version: 2.4.2-1
```

Comment:
Without this PR there's already a very strong possibility of unattended upgrades because of [this](https://github.com/saltstack-formulas/elasticsearch-formula/blob/master/elasticsearch/pkg.sls#L11):

    version: {{ major_version }}.*

I think this PR is a step in the right direction because it allows the user to control the version... you can pin a very specific version and/or use wildcards to just pin a major version, major.minor.*, etc.